### DISCUSSION TITLE
Bot Disconnected dan Gagal Scan Ulang

### DISCUSSION THREAD

**Discussion by ekoyuliawan** [kirimwa-id/forum#113] [Tanya Jawab]
Per kemarin siang bot tidak mengirim pesan dan setelah dicek statusnya disconnected/not authenticated. Namun setelah dicoba scan ulang hasilnya gagal tanpa kode error, screen Whatsapp hanya berkedip sekali. Lalu tidak terjadi apa-apa. Sudah dipastikan bukan yang multi-devices dan waktu scan juga cepat, kurang dari 10 detik. Mohon bantuannya. Terima kasih.

**Comment by ekoyuliawan** [comment]
UPDATE: Setelah saya coba ganti Whatsapp lain (nomor lain) berhasil. Apakah ada batasan tertentu ya? Atau apakah ada kemungkinan terblokir karena suatu sebab? Sedangkan untuk jumlah pesan tidak lebih dari 100 per hari. Terima kasih.

**Comment by ekoyuliawan** [comment]
UPDATE: FYI:

1. Percobaan ganti Whatsapp sebelumnya adalah menggunakan HP baru dan Whatsapp yang berbeda (BERHASIL)
2. Percobaan selanjutnya adalah menggunakan HP lama dengan nomor berbeda (GAGAL)
3. Percobaan lainnya menggunakan HP yang sama dengan versi Whatsapp Unofficial by Fouads (BERHASIL)

Belum tahu persis penyebab sebenarnya, tapi pada versi WA by Fouads adalah menggunakan core WA versi di bawahnya (tidak up-to-date).

Sampai saat ini bot masih berjalan dengan baik namun di WA unofficial by Fouads.

**Reply by rioastamal** [reply]
@ekoyuliawan, terima kasih laporannya. Whatsapp versi berapa yang terjadi kegagalan?

**Reply by ekoyuliawan** [reply]
Untuk versi Whatsapp yang gagal adalah 2.22.2.74 sedangkan yang running normal saat ini adalah versi di bawahnya yaitu 2.22.2.73. Terima kasih mas Rio.

**Comment by akmal31** [comment]
Saya kejadian juga nih, yang berhasil itu adalah yg di page perangkat tertaut ada kolom beta beta multiperangkat.
kalo ga ada kolom beta beta multiperangkatnya ga bisa
versi WA saya 2.22.4.74 versi WA yg ga bisa 2.22.7.74

**Comment by ekoyuliawan** [comment]
mas @rioastamal dan semuanya, per hari ini bot saya benar-benar sudah tidak bisa diakali lagi. Jika sebelumnya bisa didowngrade ke versi lawas dan normal kembali, sekarang sudah tidak bisa lagi karena versi berapapun langsung otomatis aktif fitur multidevice-nya.

**Reply by masbagusstwn** [reply]
harus pake unofficial maybe :)

**Reply by ekoyuliawan** [reply]
Halo mas, fyi, sejak awal saya memang sudah pakai yang Unofficial. Saya pakai WA by Fouad dari versi 9.25 sampai latest version 9.30 tidak ada yang work.

**Reply by rioastamal** [reply]
Halo @ekoyuliawan dan semua. Mohon maaf fitur ini cukup lambat dirilis. Rencana akan saya hadirkan tepat pada usia API KirimWA.id yang ke satu bulan depan. Terima kasih atas kesabarannya :)

**Reply by ekoyuliawan** [reply]
Wah, sungguh kabar yang sangat menggembirakan buat semua _kirimwa.id lovers_! Kami tidak sabar menunggunya :) Terima kasih mas Rio!
